src/test/java/core/basesyntax/RegistrationValidationTest.java

```diff
@@ -0,0 +1,148 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationValidationTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private RegistrationService registrationService;
+
+    //region Repeated Tasks
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+    //endregion
+
+    //region assertEquals Tests
+    @Test
+    void register_validUser_Ok() {
+        User validUser = new User();
+        validUser.setAge(18);
+        validUser.setLogin("validLogin");
+        validUser.setPassword("validPassword");
+        User actual = registrationService.register(validUser);
+        assertEquals(actual, validUser);

```

Comment:
The test method names should follow the convention `<methodUnderTest>_<state>_<expectedBehavior>`. For example, 'register_validUser_Ok' should be 'register_validUser_isRegistered'.